src/vue/modules/issuances-explorer/index.spec.js

```diff
@@ -0,0 +1,156 @@
+import IssuancesExplorerModule from './index'
+
+import Vuex from 'vuex'
+
+import { issuancesExplorerModule } from './store/index'
+
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+
+import { Wallet } from '@tokend/js-sdk'
+
+import * as Api from './_api'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+describe('Issuances explorer module', () => {
+  const props = {
+    config: {
+      horizonUrl: 'https://test.api.com',
+    },
+    wallet: new Wallet(
+      'test@mail.com',
+      'SCPIPHBIMPBMGN65SDGCLMRN6XYGEV7WD44AIDO7HGEYJUNDKNKEGVYE',
+      'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ',
+      '4aadcd4eb44bb845d828c45dbd68d5d1196c3a182b08cd22f05c56fcf15b153c'
+    ),
+    shouldUpdate: false,
+  }
+
+  let store
+  let wrapper
+
+  beforeEach(() => {
+    store = new Vuex.Store({
+      modules: { 'issuances-explorer': issuancesExplorerModule },
+    })
+
+    wrapper = shallowMount(IssuancesExplorerModule, {
+      store,
+      localVue,
+      propsData: props,
+    })
+  })
+
+  describe('created hook', () => {
+    it('initializes the API', () => {
```

Comment:
here are the same comments about naming test-cases. These messages should describe exactly what you test - if you test that method should call another method - that's what should the message tell us

Reply:
Updated